### PR TITLE
cloud: add doc for api v1beta1

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

as title

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## September 28, 2023
+
+**API changes**
+
+// TBD
```
Comment (ran-huang) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- Introduce a TiDB Cloud Billing API endpoint to retrieve the bill for the given month of a specific organization.  For more information, refer to the [API documentation (v1beta1)](https://docs.pingcap.com/tidbcloud/api/v1beta1).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## September 28, 2023
+
+**API changes**
+
+- Introduce a TiDB Cloud Billing API endpoint to retrieve the bill for the given month of a specific organization. For more information, refer to the [API documentation (v1beta1)](https://docs.pingcap.com/tidbcloud/api/v1beta1).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:15]: 是否需要链接到 https://docs.pingcap.com/tidbcloud/api/v1beta1#tag/Billing。

Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- Introduce a TiDB Cloud Billing API endpoint to retrieve the bill for the given month of a specific organization. 

    This Billing API endpoint is released in TiDB Cloud API v1beta1, which is the latest API version. For more information, refer to the [API documentation (v1beta1)](https://docs.pingcap.com/tidbcloud/api/v1beta1#tag/Billing).
```

File: tidb-cloud/api-overview.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ summary: Learn about what is TiDB Cloud API, its features, and how to use API to
 
 > **Note:**
 >
-> [TiDB Cloud API](https://docs.pingcap.com/tidbcloud/api/v1beta) is in beta.
+> TiDB Cloud API is in beta.
 
 The TiDB Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that provides you with programmatic access to manage administrative objects within TiDB Cloud. Through this API, you can automatically and efficiently manage resources such as Projects, Clusters, Backups, Restores, and Imports.
```
Comment (Oreoxmt) [inline tidb-cloud/api-overview.md:12]:
```suggestion
The TiDB Cloud API is a [REST interface](https://en.wikipedia.org/wiki/Representational_state_transfer) that provides you with programmatic access to manage administrative objects within TiDB Cloud. Through this API, you can automatically and efficiently manage resources such as Projects, Clusters, Backups, Restores, Imports, and Billings.
```

Comment (Oreoxmt) [reply tidb-cloud/api-overview.md:12]: L20:

```
To start using TiDB Cloud API, refer to the following resources in TiDB Cloud API Documentation:
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**